src/index.html

```diff
@@ -8,6 +8,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    
+    <div class="stars stars--1">
+      <div class="stars__star stars--N"></div>
```

Comment:
Не по БЕМ, не використовуємо uppercase в іменах класів і клас з модифікатором має дублювати той, що вже є в елемента плюс --модифікатор
```suggestion
      <div class="stars__star stars__star--active"></div>
```